tests/home/home.spec.ts

```diff
@@ -1,6 +1,7 @@
 import { test } from '../fixtures';
 
 import { SearchOption } from '../models';
+import { HomePage } from '../page-objects/home-page';
```

Comment:
Is the `HomePage` class being used anywhere in here?

Reply:
will remove this unused import